tools/pa11y.js

```diff
@@ -0,0 +1,36 @@
+const pa11y = require('pa11y');
+const reporter = require('pa11y-reporter-cli'); // As pa11y 5 stabilizes, we can pull this off pa11y
+
+// @TODO dynamically retrieve pl data from config.js.
+// An Array of URLs for pa11y to test.
+const urls = [
```

Comment:
For shits and giggles, let's:

```js
const url = require('url');

const plPath= 'http://0.0.0.0:8080/pl';
const testPaths= [
  '/patterns/00-protons-demo-borders/00-protons-demo-borders.html',
  // ...
];
// ...
const pallyPaths = testPaths.map(path => pa11y(url.resolve(plPath, path), options));
// ...
```
This brings over a couple of best practices around DRY, using proper url concatenation using built-in node functions